GPU/Common/ShaderId.cpp

```diff
@@ -119,6 +131,12 @@ void ComputeVertexShaderID(ShaderID *id_out, u32 vertType, bool useHWTransform)
 
 		id.SetBit(VS_BIT_NORM_REVERSE, gstate.areNormalsReversed());
 		id.SetBit(VS_BIT_HAS_TEXCOORD, hasTexcoord);
+
+		id.SetBit(VS_BIT_BEZIER, doBezier);
```

Comment:
I think I'd like to have an if (g_Config.bHardwareTesselation) around these, to avoid a little bit of work here if disabled.